src/test/java/com/revature/studyforce/flashcard/integration/VoteIntegrationTest.java

```diff
@@ -0,0 +1,162 @@
+package com.revature.studyforce.flashcard.integration;
+
+import com.revature.studyforce.flashcard.controller.VoteController;
+import com.revature.studyforce.flashcard.model.Answer;
+import com.revature.studyforce.flashcard.model.Flashcard;
+import com.revature.studyforce.flashcard.repository.AnswerRepository;
+import com.revature.studyforce.flashcard.repository.FlashcardRepository;
+import com.revature.studyforce.flashcard.repository.VoteRepository;
+import com.revature.studyforce.flashcard.service.VoteService;
+import com.revature.studyforce.user.model.Authority;
+import com.revature.studyforce.user.model.User;
+import com.revature.studyforce.user.repository.UserRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Test class for VoteController {@link VoteController}
+ * @author Nick Zimmerman
+ */
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestPropertySource(locations = "classpath:test-application.properties")
+class VoteIntegrationTest {
+
+    @Autowired
+    private VoteRepository voteRepository;
+    @Autowired
+    private UserRepository userRepository;
+    @Autowired
+    private AnswerRepository answerRepository;
+    @Autowired
+    private VoteService voteService;
+    @Autowired
+    private FlashcardRepository flashcardRepository;
+    @Autowired
+    private VoteController controller;
+
+
+    @Test
+    void givenVoteDTO_whenCreateAnswerVote_shouldReturnVote() throws Exception {
+        User u = new User(0,"jesus.christ@revature.com","password","Jesus","Christ",true,false,false, Authority.USER, Timestamp.valueOf(LocalDateTime.now()),Timestamp.valueOf(LocalDateTime.now()));
```

Comment:
Using outdated user model, please correct to user model with 1 name field and no password field